index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+   
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel = "icon" href =  "pictures/bxs-radiation.svg" type = "image/x-icon">
+   
+</head>
+<body>
+  <section class="prinsipal">
+    <header class="navbar">
+      <p class="name color" > <a class="nameanchor" href="http://">John Doe</a> </p>
+      <img class="bxicon" src="pictures/menu.svg" alt="">
+    </header>
+
+    <h1 class="wilcome">Hey There. I'm Eldon</h1>
+    <h2 class="ocupacion">I'am a Software Developer</h2>
+    <p class="parafo1">i can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you meed coded, don't hesitate to contact me.</p>
+  </section>
+<section class="s-media">
+    <div class="cat media"><img src="pictures/cat.svg" alt=""></div>
+    <div class="in media"><img src="pictures/linkedin.svg" alt=""></div>
+    <div class="v media"><img src="pictures/victori.svg" alt=""> </div>
+    <div class="twitter media"><img src="pictures/twitter2.svg" alt=""></div>
+    <div class="m media"><img src="pictures/M.svg" alt=""></div>
```

Comment:
- [ ] Please for your icons make sure to follow the example provided by the previous reviewer @Favourezeugwa, because this is a list of items so you should wrap them inside a `ul` tag with `li`'s.  Like this 👇️

```
      <ul class="s-media">
        <li class="cat media"><a href=""><img src="/pictures/cat.svg" alt=""></a></section>
        <li class="in media"><a href=""><img src="/pictures/linkedin.svg" alt=""></a></li>
        <li class="v media"><a href=""><img src="/pictures/victori.svg" alt=""></a></li>
        <li class="twitter media"><a href=""><img src="/pictures/twitter2.svg" alt=""></a></li>
        <li class="m media"><a href=""><img src="/pictures/M.svg" alt=""></a></li>
      </ul>
```